config/http.js

```diff
@@ -169,6 +170,22 @@ module.exports.http = {
       return next();
     },
 
+    redirectNoCacheHeaders: function (req, res, next) {
+      const originalRedirect = res.redirect;
+
+      // Patch the redirect function so that it sets the no-cache headers
+      res.redirect = function(location) {
+
+        res.set('Cache-Control', 'no-cache, no-store, must-revalidate');
```

Comment:
The `Cache-Control` value is different from the other places this header is set.
What's the reasoning for choosing these values?

- https://github.com/redbox-mint/redbox-portal/blob/develop/config/http.js#L181
- https://github.com/redbox-mint/redbox-portal/blob/develop/api/policies/noCache.js#L21
- https://github.com/redbox-mint/redbox-portal/blob/develop/core/src/CoreController.ts#L291

Reply:
I think it just comes down to different developers putting them based on different sources at different times. We'll do a full review of this in one of the upcoming releases.